installer/tue-install-impl.bash

```diff
@@ -280,8 +259,10 @@ function _try_branch_git
         tue-install-error "Invalid _try_branch_git: needs two arguments (repo and branch)."
     fi
 
-    tue-install-debug "git -C $1 checkout $2"
-    _try_branch_res=$(git -C "$1" checkout "$2" 2>&1) # This is a "global" variable from tue-install-git
+    repo="$1"
+    branch="$2"
```

Comment:
Use local variables?

Remove quotes? I don't think these are needed. We don't use them in "arg parsing" of other functions.